content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -1,44 +1,56 @@
 ---
-title: Intro to onchain development
+title: Intro to Solana Onchain Development
 objectives:
   - Understand how Solana onchain programs work
-  - Describe the structure and operation of onchain programs
+  - Know about the structure and operation of Solana programs
   - Build a basic program
 description:
-  "How onchain programs (often called 'smart contracts') work on Solana."
+  "Discover how onchain programs ( often called 'smart contracts') work on
+  Solana and learn to build your own."
 ---
 
 ## Summary
 
-- **Onchain programs** are programs that run on Solana.
-- Programs have one or more **instruction handlers**. Instruction handlers are
-  functions that process the **instructions** in Solana transactions.
-- Instruction handlers write their data to Solana **accounts** and then either
-  succeed or fail.
-- Solana programs are most commonly written in **Rust**, typically with the
-  **Anchor** framework.
-- Anchor programs have IDLs, which are small JSON files describing programs.
-  IDLs can be used to automatically create JS/TS libraries for a program.
+- **Onchain programs** refers to the code that run directly on Solana.
+- These programs consist of **instruction handlers** - functions that process
+  instructions from transactions.
+- Programs interact with the blockchain by reading from and writing to Solana
+  **accounts**.
+- Solana programs are most commonly written in **Rust**, often using the
+  **Anchor** framework for simplified development.
+- Anchor generates **Interface Description Language (IDL)** files, which:
+  - Describe the program's structure and functionality
+  - Enable automatic creation of JavaScript/TypeScript client libraries
+- Solana's architecture allows for parallel execution of non-overlapping
+  transactions, contributing to its high speed and efficiency.
+- Rent is a concept in Solana where accounts must maintain a minimum balance to
+  stay alive on the blockchain.
```

Comment:
This is all great (no feedback, just saying).

Reply:
thanks